src/modules/integrator-guide/pages/data-privacy/data-privacy-console.adoc

```diff
@@ -2,74 +2,124 @@ include::ROOT:partial$variables.adoc[]
 
 = Data Privacy Console
 
-The data privacy console enables you to access your personal data.
-It provides a platform where you can make a modification, deletions, and export requests to the company concerning your data.
-These requests(deletion, correction, and export) concerning the treatment of your data are then escalated to the xref:data-privacy/data-privacy-officer.adoc[data privacy officer (DPO)].
+The data privacy console enables you to access your data.
+It provides a platform where you may submit requests to modify, delete, and export your data.
+These requests (deletion, correction, and export) concerning the treatment of your data are forwarded to the xref:data-privacy/data-privacy-officer.adoc[data privacy officer (DPO)].
 
 This feature is also enabled to allow all users to browse locations where the data is stored.
 
 .The screenshot shows the overview of the data privacy console.
 image::data-privacy/data-privacy-console.png[role="data-zoomable"]
 
-== Features of the Privacy Console
+// == Data Overview
 
-* Data Overview
-* Privacy Requests
+// Data overview allows you to review your profile information and your data stored in Corteza applications.
 
-== Data Overview
+// image::data-privacy/data-overview.png[role="data-zoomable"]
 
-Data overview allows you to review your profile information and your personal data stored in Corteza applications.
+// * Profile Information - Contact information, names, surnames, nicknames, profile picture.
 
-image::data-privacy/data-overview.png[role="data-zoomable"]
-
-* Profile Information - Contact information, names, surnames, nickname, profile picture.
-
-* Data Application - Personal data stored in your {PRODUCT_NAME} instance.
+// * Data Application - Personal data stored in your {PRODUCT_NAME} instance.
 
 == Privacy Requests
```

Comment:
I think we'll need to replace the below table with sub-sections (so my [original review comment](https://github.com/cortezaproject/corteza-docs/pull/292/files/#r997762880)).

Replace the table rows with sub-sections and provide more detailed instructions with screenshots for specific operations.

Something in the lines of:

```
== Privacy Requests
...
=== Request Correction
...
=== Request deletion
...
=== Request Export
...
...
```

Then each of the sub-sections would provide more detailed instructions like we have them [here](https://docs.cortezaproject.org/corteza-docs/2022.9/integrator-guide/authentication/mfa.html)